lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -156,9 +171,17 @@ def configure(conf)
           @dataflow_job_id = fetch_gce_metadata('instance/attributes/job_id')
           common_labels["#{DATAFLOW_SERVICE}/job_id"] = @dataflow_job_id
         end
-      end
-
-      if (@service_name != DATAFLOW_SERVICE)
+        # dataflow only uses their own labels; otherwise include GCE labels.
```

Comment:
I meant "exclusively" and rewrote the comment to make this more clear (in managed VMs case we send both compute and appengine labels, but dataflow sends only dataflow labels).